### ISSUE TITLE
AWS_REGION and ASPNETCORE_URLS env vars are not working

### ISSUE THREAD

**Issue by mvitale-kensu** [apptality/aws-ecs-cloudmap-prometheus-sd#1]
Hey, I was trying out your code (I love it so far!) but apparently these 2 env vars are not working as expected: 
```shell
AWS_REGION
ASPNETCORE_URLS
```
---
`-e AWS_REGION="eu-west-3"`
I've got this warning: 
```shell
[WRN] Failures while fetching ECS clusters: [{"Arn": "arn:aws:ecs:us-west-2:*******:cluster/cluster1", "Detail": null, "Reason": "MISSING", "$type": "Failure"}]
```
It cannot find `cluster1` because is searching in `us-west-2`. 

---
`-e ASPNETCORE_URLS="http://*:9091"`
In the logs I see the override is in place: 
```shell
[WRN] Overriding address(es) 'http://*:9091'. Binding to endpoints defined via IConfiguration and/or UseKestrel() instead.
```
But in reality it was still listening on `9001`: 
```shell
❯ docker exec -it --user root aws-ecs-cloudmap-prometheus-discovery sh -c "netstat -nap"
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
  tcp        0      0 :::9001                 :::*                    LISTEN      -
```
And
```shell
❯ docker exec -it --user root aws-ecs-cloudmap-prometheus-discovery sh -c "curl localhost:9091/prometheus-targets"
curl: (7) Failed to connect to localhost port 9091 after 0 ms: Could not connect to server
```
---
To workaround these issues I had to set the values directly `appsettings.json`:
```json
  "AWS": {
    "Region": "eu-west-3"
  },
  "Kestrel": {
    "EndPoints": {
      "Http": {
        "Url": "http://*:9091"
      }
    }
  },
```
After updating `appsettings.json` and launching the application, everything works as expected.

**Comment by alex-tsbk** [comment]
@mvitale-kensu thank you for the feedback. I'll take a look at this on the weekend - Sept 7-8 and will updated here!

**Comment by alex-tsbk** [comment]
@mvitale-kensu if you set the following environment variables (case sensitive):
```sh
...
-e AWS_Region=eu-west-3 \
-e Kestrel_EndPoints_Http_Url="http://*:9091" \
...
```
does that address the issue? Can  you please try the above and let me know?

**Comment by mvitale-kensu** [comment]
Hey, I've tried with: 
```shell
-e AWS_Region=eu-west-3 \
-e Kestrel_EndPoints_Http_Url="http://*:9091" \
```
And I've got the same issue.

**Comment by apptality-rs** [comment]
@mvitale-kensu both issues are now fixed and verified. Please give both a try:
```
-e AWS_REGION="eu-west-3"
-e ASPNETCORE_URLS="http://*:9091"
```
should now work as expected. Fixed in version 1.0.3 (make sure you force re-pull if you're using :latest tag)

**Comment by mvitale-kensu** [comment]
Works good, thanks!
